test-app/cdk.context.json

```diff
@@ -0,0 +1,5 @@
+{
+  "acknowledged-issue-numbers": [
+    19179
```

Comment:
Hmm what's this?

Reply:
CDK was complaining, this acknowledges the "risk" so the CLI stops complaining.